app/views/unattended/kickstart/PXEGrub2.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+kind: PXEGrub2
+name: Kickstart default PXEGrub2
+oses:
+- CentOS
+- Fedora
+- RedHat
+-%>
+# This file was deployed via '<%= template_name %>' template
+<%= snippet 'kickstart_cmdline' -%>
+
+set default=0
+set timeout=<%= @loader_timeout %>
+
+menuentry '<%= template_name %>' {
```

Comment:
Yeah, it's because we set this to attribute. Changing the way how `template_name` helper works.